src/main/kotlin/com/autonomousapps/extension/Issue.kt

```diff
@@ -3,9 +3,9 @@
 package com.autonomousapps.extension
 
 import org.gradle.api.InvalidUserDataException
+import org.gradle.api.artifacts.MinimalExternalModuleDependency
 import org.gradle.api.model.ObjectFactory
 import org.gradle.api.provider.Provider
-import org.gradle.api.provider.SetProperty
```

Comment:
After rebasing this import was unused. I think maybe it was missed in a previous refactor?